compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -820,12 +820,14 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
         let mut llargs = Vec::with_capacity(arg_count);
 
         // Prepare the return value destination
-        let ret_dest = if target.is_some() {
-            let is_intrinsic = intrinsic.is_some();
-            self.make_return_dest(bx, destination, &fn_abi.ret, &mut llargs, is_intrinsic)
-        } else {
-            ReturnDest::Nothing
-        };
+        let ret_dest = self.make_return_dest(
+            bx,
+            destination,
+            &fn_abi.ret,
+            &mut llargs,
+            intrinsic.is_some(),
+            target.is_some(),
```

Comment:
... you could go further, and pass in `&target` (rather than `target.is_some()` (and maybe even `&intrinsic` too)...

... but maybe this is me just trying to get this call expression short enoguh so that it fits on one line again... :)

Reply:
(and then in other follow-up code where you pass false, you'd need to instead pass `&None`. I don't know, maybe its not a net win. I just treat a collection of boolean parameters as a red flag for readability.)